core/src/test/java/com/sequenceiq/cloudbreak/service/cluster/RepositoryConfigValidationServiceTest.java

```diff
@@ -68,8 +74,7 @@ public void testValidateForRequestsWithNullValuedFields() {
 
     @Test
     public void testValidateForAmbariBaseUrlWhenTheUrlIsReachable() {
-        doReturn(client).when(underTest).createRestClient();
-        when(client.target(anyString()).request().get()).thenReturn(response);
+        when(client.target(anyString()).request().head()).thenReturn(response);
```

Comment:
This behavior configuration could be moved from every tests into the setup() method.